app/dependencies.php

```diff
@@ -152,6 +152,20 @@ function (ContainerInterface $c): Auth\DonationPublicAuthMiddleware {
             return new Client\Mailer($settings, $c->get(LoggerInterface::class));
         },
 
+        Client\Stripe::class => function (ContainerInterface $c): Client\Stripe {
+
+            $useStubStripe = (getenv('APP_ENV') !== 'production' && getenv('BYPASS_PSP'));
```

Comment:
For now this is the same logic as we have in PSPStubber that was used for recent load tests. But I think it would make sense to adjust it so it can use a query param and we can have manual QA going on at the same time as load tests.